README.md

```diff
@@ -3,15 +3,16 @@
 > In this project I built my personal portfolio , It is a Responsive webpage so you can view it on your mobile or on a Desktop version , The webpage consists of 5 main sections: header, body-section, works-section, about myself-section and finally contact-me section.
```

Comment:
Kindly check your page. Is overlapping in the ``x-axis``:
- Contains a ``div`` tags more bigger than the 100% of your `` body width ``.
<img width="545" alt="Screen Shot 2021-09-27 at 5 33 40" src="https://user-images.githubusercontent.com/22258274/134892678-66104964-276e-4975-95b0-a0f9c2bb5691.png">